src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/unit_tests/controllers/web/ReportControllerTests/ListReportsTests.kt

```diff
@@ -0,0 +1,93 @@
+package org.vaccineimpact.orderlyweb.tests.unit_tests.controllers.web.ReportControllerTests
+
+import com.nhaarman.mockito_kotlin.any
+import com.nhaarman.mockito_kotlin.doReturn
+import com.nhaarman.mockito_kotlin.mock
+import org.assertj.core.api.AssertionsForClassTypes.assertThat
+import org.junit.Ignore
+import org.junit.Test
+import org.vaccineimpact.orderlyweb.*
+import org.vaccineimpact.orderlyweb.controllers.web.ReportController
+import org.vaccineimpact.orderlyweb.db.repositories.ReportRepository
+import org.vaccineimpact.orderlyweb.models.ReportVersionWithDescLatest
+import java.time.Instant
+
+class ListReportsTests
```

Comment:
Can we have an integration test too?

Reply:
Sorry - the existence of integration tests (as opposed to Selenium test) completely passed me by! I've added one now.